pkg/io/storage/redis_test.go

```diff
@@ -13,8 +14,9 @@ import (
 func TestRedisVault_SaveData(t *testing.T) {
 	key := "key"
 	encodedValue := "value"
-	rdb := redis.NewClient(&redis.Options{Addr: "localhost:6379", Password: "", DB: 0})
 	t.Run("success", func(t *testing.T) {
+		rdb := redis.NewClient(&redis.Options{Addr: "localhost:6379", Password: "", DB: 0})
+		defer disconnectRDB(rdb)
```

Comment:
Why do you need to create a client every test case? It is more efficient and readable to reuse the only client. Or what was your point?